docs/source/ufs_utils.rst

```diff
@@ -375,11 +375,11 @@ Program inputs and outputs
 
       * grid file - the "grid" file from the make_hgrid or regional_esg programs  - CRES_grid.tile#.nc - (NetCDF)
       * orography file - the orography file including the 'inland' flag record from the inland program - oro.CRES.tile#.nc (NetCDF)
-      * lake status code file - One of the following files. (located in `./fix/fix_orog <https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#fix/fix_orog/>`_). See GlobalLakeStatus.txt for a description of the file format.
+      * lake status code file - One of the following files. (located in `./fix/fix_orog <https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/fix/fix_orog/>`_). See GlobalLakeStatus.txt for a description of the file format.
```

Comment:
Some LakeStatus and LakeDepth files are not on AWS. Does EPIC maintain these files?

Reply:
Nope, we maintain them. Forgot to sync to AWS while syncing elsewhere...the new `orog/20240917` is syncing to S3 now. Probably be done in the next 15mins.